sdk/src/main/java/com/deploygate/sdk/DeployGateInitializeCallback.java

```diff
@@ -0,0 +1,19 @@
+package com.deploygate.sdk;
+
+/**
+ * A callback interface to receive DeployGate service initialization events.Implement this and pass to
+ * {@link DeployGate#registerInitializeCallback(DeployGateInitializeCallback)} to listen.
```

Comment:
It would be hard to maintain comments if you embedded the detailed usages. `see` annotation is enough.

```suggestion
 * A callback interface to receive DeployGate service initialization events.
```